readthedocs/vcs_support/backends/git.py

```diff
@@ -151,27 +161,22 @@ def parse_branches(self, data):
               origin/release/2.0.0
               origin/release/2.1.0
         """
-        clean_branches = []
-        # StringIO below is expecting Unicode data, so ensure that it gets it.
-        if not isinstance(data, str):
-            data = str(data)
-        raw_branches = csv.reader(StringIO(data), delimiter=' ')
-        for branch in raw_branches:
-            branch = [f for f in branch if f != '' and f != '*']
```

Comment:
`'*'` and `''` are excluded because of the regex (`\w` doesn't match `'*'`, so it that split things properly then this would be okay.